src/runtime/pay-flow.js

```diff
@@ -71,21 +76,23 @@ export class PayStartFlow {
    * @return {!Promise}
    */
   start() {
+    // Add the 'publicationId' key to the subscriptionRequestKeys_ object.
+    const swgPaymentRequest =
+        Object.assign({}, this.subscriptionRequestKeys_, {
```

Comment:
We also will need to handle that enum question I mentioned above.

Reply:
done